src/main/java/tutorease/address/model/Model.java

```diff
@@ -115,4 +115,9 @@ public interface Model {
      * Returns size of lesson schedule.
      */
     int getLessonScheduleSize();
+
+    /**
+     * Deletes all the lessons of a particular student
+     */
```

Comment:
perhaps right this in the javadoc form with the parameter.
```suggestions
/**
* Deletes all the lessons of a particular student
*
* @param student The person object to look up.
*/